aten/src/ATen/native/native_functions.yaml

```diff
@@ -177,6 +177,10 @@
 
 - func: _assert_tensor_metadata(Tensor a, SymInt[]? size=None, SymInt[]? stride=None, ScalarType? dtype=None) -> ()
 
+- func: sym_constrain_range(Scalar size, int? min=None, int? max=None) -> ()
+  dispatch:
+    CPU: sym_constrain_range_cpu
```

Comment:
Can you also add dummy CUDA implementation for it?

Reply:
Will do